src/components/port-picker/PortsInput.vue

```diff
@@ -91,7 +91,7 @@ export default {
     },
 };
 </script>
-<style scoped>
+<style scoped lang="less">
```

Comment:
Do we need `lang="less"`here? Don't think we are using any features from less in vue parts